solc/CommandLineParser.cpp

```diff
@@ -718,7 +718,7 @@ General Information)").c_str(),
 		(CompilerOutputs::componentName(&CompilerOutputs::signatureHashes).c_str(), "Function signature hashes of the contracts.")
 		(CompilerOutputs::componentName(&CompilerOutputs::natspecUser).c_str(), "Natspec user documentation of all contracts.")
 		(CompilerOutputs::componentName(&CompilerOutputs::natspecDev).c_str(), "Natspec developer documentation of all contracts.")
-		(CompilerOutputs::componentName(&CompilerOutputs::metadata).c_str(), "Combined Metadata JSON whose Swarm hash is stored on-chain.")
+		(CompilerOutputs::componentName(&CompilerOutputs::metadata).c_str(), "Request the combined Metadata JSON whose Swarm hash is stored on-chain. If used with --output-dir the metadata will be written in that directory, one file per contract each named \"<ContractName>_meta.json\", otherwise it is displayed on stdout.")
```

Comment:
Now this is inconsistent with how all other components are described. They are all listed as things, but this one is now an action. I'd keep this part the way it was:
```suggestion
		(CompilerOutputs::componentName(&CompilerOutputs::metadata).c_str(), "Combined metadata JSON whose Swarm hash is stored on-chain. If used with --output-dir the metadata will be written in that directory, one file per contract each named \"<ContractName>_meta.json\", otherwise it is displayed on stdout.")
```

If you really want to change it, I think you should change it for all of them. But the way it is now is more succinct and still comprehensible so personally I'd keep it.